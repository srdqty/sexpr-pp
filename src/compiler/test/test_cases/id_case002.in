#| Individual expressions have independent global variable name mappings |#
x
y
z
(+ x y z)
(lambda ()
  x
  y
  z
  (+ x y z))
