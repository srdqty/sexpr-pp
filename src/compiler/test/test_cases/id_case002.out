(GlobalRef 0)
(GlobalRef 0)
(GlobalRef 0)
(App (GlobalRef 0) (ArgCount 3) (Args (GlobalRef 1) (GlobalRef 2) (GlobalRef 3)))
(FunFixArity (Arity = 0) (GlobalRef 0) (GlobalRef 1) (GlobalRef 2) (App (GlobalRef 3) (ArgCount 3) (Args (GlobalRef 0) (GlobalRef 1) (GlobalRef 2))))
